interfaces/builtin/opengl.go

```diff
@@ -53,8 +54,13 @@ const openglConnectedPlugAppArmor = `
 # Support reading the GLVND EGL vendor files
 /var/lib/snapd/lib/glvnd/ r,
 /var/lib/snapd/lib/glvnd/** r,
+/var/lib/snapd/hostfs/usr/share/glvnd/egl_vendor.d/ r,
```

Comment:
How does /usr/share/glvnd/egl_vendor.d/ looks like for tegra drivers? what files are in there?

Reply:
On Xavier NX, the directory currently have a single file `10-nvidia.json`, however it can have multiple of those and the initial integer in the name represents the priority.
```
root@jetson-xavier-nx-devkit:~# cat /usr/share/glvnd/egl_vendor.d/10-nvidia.json 
{
    "file_format_version" : "1.0.0",
    "ICD" : {
        "library_path" : "libEGL_nvidia.so.0"
    }
}
```